### ISSUE TITLE
Redrawing doesn't trigger animations for updated series with unchanged y-axis

### ISSUE THREAD

**Issue by gabestrom** [highcharts/highcharts#14093]
Context:
I have created a custom filter that selects a single series to be displayed and sets the rest to be hidden using series.setVisible(false,false)
For my chart I am using series.setData() to update our chart and then chart.redraw() to separately draw. 

Issue:
I am wondering if this is to be considered a bug or is there a solution that I am missing. 

#### Expected behaviour
chart.redraw() triggers animation on new series being charted with no change to axis

#### Actual behaviour
I'm noticing that on chart.redraw() the animation is not always firing for line chart if the y-axis does not change. 
This is occurring when I filter between two series that share the **same y-axis range.** 
However I am seeing the animations when I switch between two series that have **different y-axis ranges.** 

#### Product version
highcharts: "8.1.2"
angular: 9.1.12
typescript: "3.8.3"

#### Affected browser(s)
Brave: Version 1.11.104 Chromium: 84.0.4147.105 (Official Build) (64-bit)

**Comment by pawelfus** [comment]
Hi @gabestrom 

Thanks for the detailed description! What kind of animation do you see? Is that initial animation (from left to right)? Or it's animating from old position to new one? For example using this demo: https://www.highcharts.com/demo/line-basic (click on legend items to hide&show series) sometimes we get animation (when extremes do change) and sometimes we don't (extremes are the same). It's hard to say more without live demo. Do you use multiple yAxes? Or just just one shared across all series? You mentioned `setVisible()` but also `setData()` and `redraw()` - which one do you really use? All three when filtering series? Note that `setData()` can animate points differently, depending on number of points ([docs](https://api.highcharts.com/class-reference/Highcharts.Series#setData) => `updatePoints` parameter).

In short, live demo would be great :) Otherwise we can only guess how can we help you.

**Comment by sebastianbochan** [comment]
Closed due to inactivity.
